components/ai_chat/content/browser/ai_chat_tab_helper.h

```diff
@@ -137,12 +169,19 @@ class AIChatTabHelper : public content::WebContentsObserver,
   raw_ptr<AIChatMetrics> ai_chat_metrics_;
 
   bool is_same_document_navigation_ = false;
-  int64_t pending_navigation_id_;
+  int pending_navigation_id_;
   bool is_pdf_a11y_info_loaded_ = false;
   uint8_t check_pdf_a11y_tree_attempts_ = 0;
-  GetPageContentCallback pending_get_page_content_callback_;
 
   raw_ptr<content::WebContents> inner_web_contents_ = nullptr;
+
+  // TODO(petemill): Use signal to allow for multiple callbacks
+  ConversationHandler::GetPageContentCallback
```

Comment:
It doesn't need to IMO because that is actually handled at the higher level (`AssociatedContentDriver`). However, this class doesn't know that. For non-pdf page's, `GetPageContent` can handle multiple calls at the same time (though it doesn't queue them, they'll just run independently). But for PDF documents, the existing `pending_get_page_content_callback_` will get called with no content and only the latest call will be used. But it's not an issue with the way `GetPageContent` is called right now. So, the TODO is there as a direction in case `AIChatTabHelper::GetPageContent` gets used in other ways in the future.